lib/queue.js

```diff
@@ -1,165 +0,0 @@
-"use strict"
```

Comment:
This file isn't needed anymore, we moved to using async/await which has the queueing ability